### DISCUSSION TITLE
How to initialize the resources of App during the installation?

### DISCUSSION THREAD

**Discussion by KikkiZ** [tauri-apps/tauri#10073] [Q&A]
I want to  run some initialize code to generate some config files and resources that the app needs  automatically  when the user launches/installs the app for the first time. 

I didn't find an introduction to it in the documentation. I wonder if there is any way to customize the installation process.

**Comment by biplab5464** [comment]
Looking for the same thing, if there's a way to run a specific function at installation time to make some configuration changes.
